pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -56,17 +56,30 @@ func NewClient(kubeConfigPath string, requestTimeout int64) (*Client, error) {
 
 	_, err = extensionsClientset.CustomResourceDefinitions().Get("operators.kudo.dev", v1.GetOptions{})
 	if err != nil {
-		return nil, errors.WithMessage(err, "operators")
+		// timeout is not a wrappable error, timeout is an underlying issue that is NOT CRD specific, there is no value in wrapping or converting as well.
+		// best to provide the actual error for proper reporting.
+		if os.IsTimeout(err) {
```

Comment:
I understand that we don't wrap the timeout error but to what end? Where do you do anything with it?

Reply:
The intention is to do something with it :).   I was wrongly of the opinion that this was such a simple and not debatable change that I could land this quickly and move on to using it.